clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchFieldMapper.scala

```diff
@@ -20,6 +20,16 @@ private[dataaccess] sealed abstract class ElasticsearchFieldMapper(
   def stringToDefinition(fieldType: Type): String => FieldDefinition
 
   def valueToQuery(fieldName: String, fieldType: Type): Any => QueryDefinition
+
+  private def is[A: TypeTag](tpe: Type): Boolean = {
+    /*
+     * Elasticsearch has no notion of an array field; instead, any
+     * field can hold 1+ values of the same type.
+     */
+    tpe =:= typeOf[A] || tpe =:= typeOf[Option[A]] || tpe =:= typeOf[Seq[A]] || tpe =:= typeOf[
```

Comment:
For the `Seq` types, I think we want `<:<` instead of `=:=` so we catch subtypes too.